### PR TITLE
chore: update buildkit to 1.13.1

### PR DESCRIPTION
**Pull request by sozercan**
Describe the changes in this pull request using active verbs such as _Add_, _Remove_, _Replace_ ...

- updates buildkit to 0.13.1
- debug uses plain mode

Closes #<_issue_ID_>

### CODE REVIEW

File: pkg/patch/patch.go [hunk 1]
```diff
@@ -231,13 +231,24 @@ func patchWithContext(ctx context.Context, ch chan error, image, reportFile, pat
...
+	if cn, err := console.ConsoleFromFile(os.Stderr); err == nil {
+		c = cn
+	}
+
+	mode := progressui.AutoMode
+	if log.GetLevel() >= log.DebugLevel {
+		mode = progressui.PlainMode
+	}
+
+	display, err := progressui.NewDisplay(c, mode, progressui.WithPhase(c.Name()))
```
Comment (cpuguy83) [inline pkg/patch/patch.go:244]: Not sure `c.Name()` is the right thing here.
Can probably leave it without a phase set.

**Review by cpuguy83 — CHANGES_REQUESTED** [review]

File: pkg/patch/patch.go [hunk 2]
```diff
@@ -231,13 +231,22 @@ func patchWithContext(ctx context.Context, ch chan error, image, reportFile, pat
...
+		c = cn
+	}
+	mode := progressui.AutoMode
+	if log.GetLevel() >= log.DebugLevel {
+		mode = progressui.PlainMode
+	}
+	display, err := progressui.NewDisplay(c, mode)
+	if err != nil {
+		return err
+	}
```
Comment (cpuguy83) [inline pkg/patch/patch.go:245]:
```suggestion
	out := os.Stderr
	if cf, _ := console.ConsoleFromFile(os.Stderr); cf != nil {
		out = cf
	}
	mode := progressui.AutoMode
	if log.GetLevel() >= log.DebugLevel {
		mode = progressui.PlainMode
	}
	display, err := progressui.NewDisplay(out, mode)
	if err != nil {
		return err
	}
```

**Review by cpuguy83 — CHANGES_REQUESTED** [review]

File: pkg/patch/patch.go (hunk 2 above)
Comment (sozercan) [reply pkg/patch/patch.go]: @cpuguy83 i removed `ConsoleFromFile` completely, wdyt?

Comment (cpuguy83) [reply pkg/patch/patch.go]: I think if you do that it will never do TTY mode?

Comment (sozercan) [reply pkg/patch/patch.go]: tested and works with tty (output is glitchy for whatever reason on my terminal, but it was the same before)

**Review by cpuguy83 — APPROVED** [review]

**Pull request merged**